source/agora/consensus/validation/Transaction.d

```diff
@@ -192,30 +212,31 @@ unittest
         ]
     );
 
-    secondTx.inputs[0].signature = key_pairs[0].secret.sign(hashFull(secondTx)[]);
+    secondTx.inputs[0].unlock = signUnlock(key_pairs[0], secondTx);
 
     // It is validated. (the sum of `Output` < the sum of `Input`)
-    assert(secondTx.isValid(storage.getUTXOFinder(), Height(0), checker),
+    assert(secondTx.isValid(engine, storage.getUTXOFinder(), Height(0), checker),
```

Comment:
// It is validated. (the sum of `Output` < the sum of `Input`)
The description above is invalid.

Reply:
It may not be related to this PR, but it's an old document.